legend-pure-ide-light/src/main/java/org/finos/legend/pure/ide/light/api/concept/RenameConceptUtility.java

```diff
@@ -23,8 +23,8 @@
 public class RenameConceptUtility
 {
     private static final String CONCRETE_FUNCTION_DEFINITION = "ConcreteFunctionDefinition";
-    private static final Pattern IDENTIFIER_REGEX = Pattern.compile("^([a-zA-Z0-9_][a-zA-Z0-9_$]*)");
-    private static final Pattern SIGNATURE_SUFFIX_REGEX = Pattern.compile("_([a-z]|[A-Z]|[0-9]|_|$)*_(.)*", Pattern.DOTALL);
+    private static final Pattern IDENTIFIER_REGEX = Pattern.compile("^([\\w_][\\w_$]*)");
+    private static final Pattern SIGNATURE_SUFFIX_REGEX = Pattern.compile("_[\\w_$]+_");
```

Comment:
This is definitely not equivalent to the previous regex. I cannot, however, comment on the correctness of the change.

Reply:
@kevin-m-knight-gs I don't know why we have the second pattern like `"_([a-z]|[A-Z]|[0-9]|_|$)*_(.)*"` for function signature, what is the final `(.)*` for? So I deleted it

Reply:
I don't even really understand what it's checking for. Is it just there to check that something is a function name by checking that the end of it looks maybe like the kinds of names we generate in Pure?